corehq/apps/builds/models.py

```diff
@@ -239,6 +251,16 @@ def get_menu(self, application_version=None):
         else:
             return self.menu
 
+    @classmethod
+    def j2me_enabled_configs(cls):
+        """
+        Fetch labels for CommCareBuilds compatible with J2ME
+        """
+        j2me_enabled_configs = []
+        for version_config in cls.fetch().menu:
+            if version_config.j2me_enabled:
+                j2me_enabled_configs.append(version_config.label)
+        return j2me_enabled_configs
```

Comment:
(same as above, list comprehension would be better here so that you can return the labels)